docs: rebranding changes (#3014)
chanzuckerberg/single-cell-data-portal: frontend/doc-site/05__Annotate and Analyze Your Data/5_5__Gene Sets.mdx

# Gene Sets

> CZ CELLxGENE Annotate enables CELLxGENE users to pre-load gene sets for exploration, and to create or modify gene sets while exploring data. This is particularly useful for recording cell type markers when annotating cells. This section explains how to format gene sets data for preloading of gene sets and where to find gene sets information after making changes in the explorer.

Reviewer: ```suggestion
CZ CELLxGENE Annotate enables users to pre-load gene sets for exploration, and to create or modify gene sets while exploring data. This is particularly useful for recording cell type markers when annotating cells. This section explains how to format gene sets data for preloading of gene sets and where to find gene sets information after making changes in the explorer.
```

> To read more about how to use gene sets in the CELLxGENE interface, please see [Creating a Geneset](https://docs.google.com/presentation/d/1ihFrKRSxnfFNexctOZ0EkUow9SWgEr-9TOVKvVLE8ms/edit#slide=id.gf4b1a39681_0_0)

Reviewer: ```suggestion
To read more about how to use gene sets in the CELLxGENE Annotate and Explorer interfaces, please see [Creating a Geneset](https://docs.google.com/presentation/d/1ihFrKRSxnfFNexctOZ0EkUow9SWgEr-9TOVKvVLE8ms/edit#slide=id.gf4b1a39681_0_0)
```

## Gene Sets Requirements

A gene set must have a unique name. A given gene may be included in multiple gene sets. A gene set may optionally include a description for the entire gene set, and descriptions for each gene included the gene set.

## `cellxgene` CLI Command

> Users MAY use `--gene-sets-file` `name_of_file.csv` to designate a CSV with preexisting gene sets you would like to view in CELLxGENE. The CSV MUST follow the format below.

Reviewer: ```suggestion
Users MAY use `--gene-sets-file` `name_of_file.csv` to designate a CSV with preexisting gene sets you would like to view in CELLxGENE Annotate. The CSV MUST follow the format below.
```

  </tr>
</table>

> Users MAY include additional columns. Once gene sets are edited from CELLxGENE, the additional columns will no longer be stored in the file designated using the command `--gene-sets-file`.

Reviewer: ```suggestion
Users MAY include additional columns. Once gene sets are edited from CELLxGENE Annotate, the additional columns will no longer be stored in the file designated using the command `--gene-sets-file`.
```

## `gene_set_description`

> Populating `gene_set_description` is optional. The first instance where `gene_set_description` is populated for a specific `gene_set_name` will be surfaced when a user hovers over `gene_set_name` in CELLxGENE. All other instances are ignored in subsequent rows for the same `gene_set_name`.

Reviewer: ```suggestion
Populating `gene_set_description` is optional. The first instance where `gene_set_description` is populated for a specific `gene_set_name` will be surfaced when a user hovers over `gene_set_name` in CELLxGENE Annotate. All other instances are ignored in subsequent rows for the same `gene_set_name`.
```

## `gene_symbol`

A given `gene_symbol` may only be added once to a gene set and exist as a VAR in the underlying anndata file.

## `gene_description`

> Populating `gene_description` is optional and will be surfaced when a user hovers on `gene_symbol` in CELLxGENE.

Reviewer: ```suggestion
Populating `gene_description` is optional and will be surfaced when a user hovers on `gene_symbol` in CELLxGENE Annotate or Explorer.
```
